src/components/FavoriteActivities/styles.css

```diff
@@ -0,0 +1,4 @@
+.main-content {
+    background-color: #F3F3F5 !important;
+    padding: 32px 0;
+}
```

Comment:
I think this has been a conflict resolution after this branch was updated, please revert it back.